templates/ocs-deployment/storageclass.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: {{ k8s_api_version }}
+kind: StorageClass
+metadata:
+   name: {{ storageclass_name | default('ocsci-storageclass') }}
+provisioner: ceph.rook.io/block
+allowVolumeExpansion: {{ volume_expansion | default('True') }}
+reclaimPolicy: {{ reclaimPolicy | default('Delete') }}
+parameters:
+  blockPool: {{ blockPool }}
+  clusterNamespace: {{ clusterNamespace | default('openshift-storage') }}
+  fstype: {{ fstype | default('xfs') }}
```

Comment:
same

Reply:
@vavuthu  This storageclass.yaml uses the flex provisioner(ceph.rook.io/block). Shouldn't we be adding the CSI provisioner based  storageclass.yaml files  from the rook repo's (csi/examples/rbd and csi/examples/cephfs) . Please comment. Once #58 is merged, we would not need the "ceph.rook.io/block" provisioner based SC files.

Reply:
I will make chnages to use CSI once #58 is merged

Reply:
> same

Done